pages/common/inkmake.md

```diff
@@ -0,0 +1,24 @@
+# inkmake
+
+> Exports from SVG files using Inkscape's backend.
```

Comment:
```suggestion
> Use inkfiles to export SVG files using Inkscape's backend.
```

Reply:
Maybe that makes it more clear what an inkfile actually is. I was really confused at the beginning.